web/cobrands/tfl/base.scss

```diff
@@ -277,4 +277,13 @@ ol.big-numbers>li {
     margin: 0
 }
 
+/* Safety critical reports */
+.item-list__item--safety-critical {
+    background-color: #ffb6b6;
```

Comment:
Sorry not to think about this sooner, but we must not rely only on colour to show something like this: https://www.w3.org/TR/WCAG21/#use-of-color
Can it also prefix the title with `Safety Critical` or `[SC]` or have some other non-colour way of showing it is such a report.